content/zh-cn/docs/reference/config-api/kubeconfig.v1.md

```diff
@@ -0,0 +1,922 @@
+</td>
+</tr>
+<tr><td><code>as</code><br/>
+<code>string</code>
+</td>
+<td>
+  <!--
+  Impersonate is the username to impersonate.  The name matches the flag.
+  -->
+   <p><code>Impersonate</code> 是要模仿的用户名. 名字与国旗相匹配.</p>
+</td>
+</tr>
+<tr><td><code>as-uid</code><br/>
+<code>string</code>
+</td>
+<td>
+  <!--
+  ImpersonateUID is the uid to impersonate.
+  -->
+   <p><code>ImpersonateUID</code> 就是 uid 为了冒充.</p>
+</td>
+</tr>
+<tr><td><code>as-groups</code><br/>
+<code>[]string</code>
+</td>
+<td>
+  <!--
+  ImpersonateGroups is the groups to impersonate.
+  --->
+   <p><code>ImpersonateGroups</code> 就是 要冒充的组.</p>
+</td>
+</tr>
+<tr><td><code>as-user-extra</code><br/>
+<code>map[string][]string</code>
+</td>
+<td>
+  <!--
+  ImpersonateUserExtra contains additional information for impersonated user.
+  --->
+   <p><code>ImpersonateUserExtra</code> 包含给模拟用户的额外信息.</p>
+</td>
+</tr>
+<tr><td><code>username</code><br/>
+<code>string</code>
+</td>
+<td>
+  <!---
+  Username is the username for basic authentication to the kubernetes cluster.
+  --->
+   <p><code>Username</code> 是对<code>kubernetes</code> 集群进行基本认证的用户名.</p>
+</td>
+</tr>
+<tr><td><code>password</code><br/>
+<code>string</code>
+</td>
+<td>
+  <!---
+  Password is the password for basic authentication to the kubernetes cluster.
+  --->
+   <p><code>Password</code> 是对 <code>kubernetes</code> 集群进行基本认证的密码.</p>
+</td>
+</tr>
+<tr><td><code>auth-provider</code><br/>
+<a href="#AuthProviderConfig"><code>AuthProviderConfig</code></a>
+</td>
+<td>
+  <!---
+  AuthProvider specifies a custom authentication plugin for the kubernetes cluster.
+  --->
+   <p><code>AuthProvider</code> 为 <code>kubernetes</code> 集群指定自定义身份验证插件.</p>
+</td>
+</tr>
+<tr><td><code>exec</code><br/>
+<a href="#ExecConfig"><code>ExecConfig</code></a>
+</td>
+<td>
+  <!---
+  Exec specifies a custom exec-based authentication plugin for the kubernetes cluster.
+  --->
+   <p><code>Exec</code> 指定了针对 <code>kubernetes</code> 集群的自定义基于 <code>exec</code> 的身份验证插件.</p>
+</td>
+</tr>
+<tr><td><code>extensions</code><br/>
+<a href="#NamedExtension"><code>[]NamedExtension</code></a>
+</td>
+<td>
+  <!---
+  Extensions holds additional information. This is useful for extenders so that reads and writes don't clobber unknown fields.
+  --->
+   <p><code>Extensions</code> 保存额外信息. 这对于扩展器是有用的，这样读写不会破解未知字段</p>
+</td>
+</tr>
+</tbody>
+</table>
+
+## `AuthProviderConfig`     {#AuthProviderConfig}
+    
+<!--
+**Appears in:**
+-->
+**出现在:**
+
+- [AuthInfo](#AuthInfo)
+
+<!--
+AuthProviderConfig holds the configuration for a specified auth provider.
+-->
+<p>AuthProviderConfig 保存特定认证提供商的配置.</p>
+
+
+<table class="table">
+<thead><tr><th width="30%"><!--Field-->字段</th><th><!--Description-->描述</th></tr></thead>
+<tbody>
+    
+  
+<tr><td><code>name</code><B><!--[Required]-->[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <span class="text-muted"><!--No description provided-->环境变量名称.</span></td>
```

Comment:
WOW！ How did the translator get this correct!!???

Reply:
No it didn't all 4 translator were wrong!! on this one but as I said I was also looking at other translated files 
so I already new the  exact translation for "No description provided",   "field", "Description" and "Required"
But the  thing that I missed out and regret were the Capital letter conversions.. as I can see became a repeatable mistake